inject/inject.vala

```diff
@@ -59,7 +61,18 @@ namespace Frida.Inject {
 			script_source = read_stdin ();
 		}
 
-		application = new Application (target_pid, target_name, script_path, script_source, enable_jit, enable_development);
+		Json.Node parameters = new Json.Node (Json.NodeType.OBJECT);
+		parameters.set_object (new Json.Object ()); 
+		if (parameters_str != null && parameters_str != "") {
```

Comment:
The empty string case should be an error. We can just ignore that here and let the parsing below fail.

Reply:
The empty string doesn't cause an error if you let it parse. "" is considered valid JSON in glib. 

Added a case for it, although I'm not sure it erroring is any better than just continuing on with an empty object.